### PR TITLE
Frontend API usage example

### PR DESCRIPTION
**Pull request by gregslomin**
[changed] fetch_assets endpoint to use manual token fetching insteadof django-esi decorators
[changed] frontend/package.json to use API request proxying
[added] SortedTable.js component
[added] Basic buttons to login, fetch assets, and a table to show asset results

### CODE REVIEW

File: backend/cchx/settings.py [hunk 1]
```diff
@@ -44,12 +46,14 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'corsheaders',
     'esi',
     "eve_auth",
```
Comment (Lyari-Elatha) [inline backend/cchx/settings.py]: you know what you did!

Comment (gregslomin) [reply backend/cchx/settings.py]: I FIXED IT

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,24 +1,48 @@
...
+          Header: "DPS %",
+          accessor: "dps"
+        },
+        {
+          Header: "Damage",
+          accessor: "damage"
+        },
+        {
+          Header: "Rate of Fire",
+          accessor: "rof"
```
Comment (Lyari-Elatha) [inline frontend/src/App.js:34]: missing cpu

File: frontend/src/components/SortedTable.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        <table {...getTableProps()}>
+            <thead>
+            {headerGroups.map(headerGroup => (
+                <tr {...headerGroup.getHeaderGroupProps()}>
+                    {headerGroup.headers.map(column => (
+                        // Add the sorting props to control sorting. For this example
+                        // we can add them into the header props
+                        <th {...column.getHeaderProps(column.getSortByToggleProps())}>
+                            {column.render('Header')}
+                            {/* Add a sort direction indicator */}
```
Comment (Lyari-Elatha) [inline frontend/src/components/SortedTable.js]: different comment style (consistency reeeeeeee)

**Review by Lyari-Elatha — APPROVED** [review]

**Pull request merged**